manifests/image-references

```diff
@@ -5,4 +5,8 @@ spec:
   - name: cluster-kube-scheduler-operator
     from:
       kind: "DockerImage"
-      name: "openshift/origin-cluster-kube-scheduler-operator:latest"
+      name: "docker.io/openshift/origin-cluster-kube-scheduler-operator:latest"
```

Comment:
still docker.io?

Reply:
did we start pushing these images to quay.io too?